templates/report.html

```diff
@@ -276,8 +276,16 @@ <h1 class="display-4">Results</h1>
   </div>
 </div>
 
+{% for tplan in dset.testplans %}
+
+<div class="jumbotron jumbotron-fluid">
+  <div class="container">
+    <h2 class="display-8">Testplan {{ tplan.name }} ( {{ tplan.ident }} )</h1>
```

Comment:
Could be a typo here? The closing tag is different from the opening :thinking:

Reply:
Good catch!

Reply:
That is a good catch! Thanks!

Reply:
fixed :)